pyproject.toml

```diff
@@ -84,3 +79,46 @@ sphinxcontrib-mermaid = "^0.9.2"
 [build-system]
 build-backend = "poetry.core.masonry.api"
 requires = ["poetry-core>=1.2.0"]
+
+[tool.pytest.ini_options]
+addopts = """	
+  -v
+  -p no:warnings
+  --cov=virtual_ecosystem
+  --cov-report=html:htmlcov/coverage
+  --doctest-modules --ignore=virtual_ecosystem/__main__.py
+  """
+python_files = 'test_*.py'
+testpaths = ['tests']
+
+[tool.ruff]
+target-version = "py310"
+
+[tool.ruff.lint]
+# Enable all `pydocstyle` rules, limiting to those that adhere to the
+# Google convention via `convention = "google"`, below.
+select = [
+  # "B", # flake8-bugbear
+  # "SIM", # flake8-simplify
+  "E", # pycodestyle
+  "F", # pyflakes
+  "D", # pydocstyle
+  "I", # isort
+  "UP", # pyupgrade
+]
+
+# On top of the Google convention, disable:
+ignore = [
+  "D202", # Blank line after docstring is ok
+  "D107", # Location of __init__ docstring in class not __init__"
+  "F401", # module imported but unused
```

Comment:
Are you sure you want to ignore this warning globally? I find it quite convenient to have `ruff` point out when I have unused imports (and automatically remove them). I'd just suppress it in the few places where you're doing it on purpose, if I were you.

Reply:
I think that's a 'legacy' setting.